CHANGES.md

```diff
@@ -1,3 +1,28 @@
+## edge-19-2.5
+
+* CLI
+  * Updated `linkerd check` to ensure hint URLs are displayed for RPC checks
+* Controller
+  * Updated the auto-inject admission controller webhook to respond to UPDATE
+    events of deployment workload
+  * Updated destination service to return TLS identities only when the client
+    pod is TLS-aware and is in the same controller namespace
```

Comment:
Rather than calling this the "client pod", I think it makes more sense to call it the "destination pod".